source/common/http/header_map_impl.h

```diff
@@ -53,10 +53,13 @@ class HeaderMapImpl : public HeaderMap {
    */
   bool operator==(const HeaderMapImpl& rhs) const;
 
-  // Http::HeaderMap
+  // Http::HeaderMap -- read comments there about what's copied and what's not!
```

Comment:
please delete the extra comment here. It's not done anywhere else. The general expectation is that people read the headers.